apps/condo/pages/ticket/index.tsx

```diff
@@ -135,6 +135,7 @@ export const TicketsPageContent = ({
     sortableProperties,
     useTableColumns,
     showImport = false,
+    baseQueryLoading = false,
```

Comment:
I think "base query" is not clear name for that context. Maybe add little bit more context to this prop? For example `relatedOrganizationsLoading`?

Reply:
No, just here it should be without the context of the cc.
This can also be used if we need additional requests in the condo to load tickets.